File: java/16.06.19/prime.java
import java.util.*;
public class prime
{
    public static void main(String[] argc)
    {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        int n;
        for(int j = 0; j < t; j++)
        {
            boolean flag = true;
            n = scn.nextInt();
            for (int i = 2; i*i <= n; i++)
            {
                if (n%i == 0)
                {
                    flag = false;
                    break;
                }
            }
            if (flag)
            {
                System.out.println("Prime!");
            }   
            else
            {
                System.out.println("Not Prime!");
            }         
        }
    }
}

File: cpp/16.06.19/prime.cpp
#include <iostream>
using namespace std;

int main(int argc, char** argv)
{
    int n;
    cin >> n;
    int flag = 0;
    for (int i = 2; i < n; i++)
    {
        if(n%i == 0)
        {
            flag = 1;
        }
    }
    if(flag == 0)
        cout  << "Prime!";
    else
        cout << "Not Prime";
}